plot_continuous_monitor.py

```diff
@@ -191,6 +192,16 @@ def parse_args():
     help="Whether to open the resulting graph PDFs.")
   return parser.parse_args()
 
+def get_util_for_disk(disk_utils, disk):
+  """
+  Returns the disk utilization metrics for the specified disk, given the utilization information
+  for all disks, or None if the desired disk cannot be found.
+  """
+  for disk_util in disk_utils:
+    if disk in disk_util:
+      return disk_util[disk]
+  return None
```

Comment:
The result of this function is supposed to be a dictionary, so if None is returned then there will be runtime errors when the follow-up code tries to do key lookups on None. Would it be better to just throw an exception instead of returning None?